src/conversion/model-to-view-converters.js

```diff
@@ -552,3 +516,24 @@ export function eventNameToConsumableType( evtName ) {
 
 	return parts[ 0 ] + ':' + parts[ 1 ];
 }
+
+/**
+ * Creates `span` {@link module:engine/view/attributeelement~AttributeElement view attribute element} from information
+ * provided by {@link module:engine/conversion/buildmodelconverter~VirtualSelectionDescriptor} object.
+ *
+ * @param {module:engine/conversion/buildmodelconverter~VirtualSelectionDescriptor }descriptor
+ * @return {module:engine/view/attributeelement~AttributeElement}
+ */
+export function virtualSelectionDescriptorToAttribute( descriptor ) {
```

Comment:
It's better to write `toElement` rather than `toAttribute` if you mean `AttributeElement`